content/03.results.md

```diff
@@ -39,10 +40,15 @@ More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/ne
 In 2011, the Children’s Brain Tumor Tissue Consortium, now known as the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection.
 Since the CBTN opened its protocol before molecular data were integrated into classifications, the majority of the samples within the OpenPBTA lacked molecular subtype annotations.
 Moreover, the OpenPBTA data does not feature methylation arrays which are now commonly used to perform molecular subtyping.
-Therefore, we jointly considered key genomic features of tumor entities described by the WHO in 2016, low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], as well as clinician and pathologist review to subtype 64% (1,281/2,007) of tumor biospecimens with high confidence (**Table S1**).
+Therefore, we jointly considered key genomic features of tumor entities described by the WHO in 2016, low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], as well as clinician and pathologist review to subtype 60% (641/1074) of tumor samples with high confidence (**Table S1**). <!--SAMPLECOUNT-->
+<!--QUESTION: Do we want to use the "integrated" count here or the number subtyped.
+I have used the molecular subtyped count, but the earlier part of the sentence discussed WHO, and that is what is in the summary, so I wasn't sure.
+For reference, the WHO integrated # is 630 or 59%-->
```

Comment:
Calling out this question in the text. Since we use the integrated counts in the Summary, it seems we should probably mention the integration results here as well.

Reply:
yeah we should probably make this integrated. Can you word it as "research integrated diagnosis" - I just remembered I have had this convo with some clinicians who made a point that we should be more specific since this is not a clinical integrated diagnosis.